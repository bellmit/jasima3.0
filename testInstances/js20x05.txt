# These data correspond to the instance ft20 (mt20) from:
#
# H. Fisher, G.L. Thompson (1963), Probabilistic learning combinations
# of local job-shop scheduling rules, J.F. Muth, G.L. Thompson (eds.),
# Industrial Scheduling, Prentice Hall, Englewood Cliffs, New Jersey,
# 225-251
#
# The optimal CMax is 1165

#number of machines
5

#number of routes
20

# machine orders <machine number(s)>;<setup state> (no space after or before comma/semicolon)
1	2	3	4	5
1	2	4	3	5
2	1	3	5	4
2	1	5	3	4
3	2	1	4	5
3	2	5	1	4
2	1	3	4	5
3	2	1	4	5
1	4	3	2	5
2	3	1	4	5
2	4	1	5	3
3	1	2	4	5
1	3	2	4	5
3	1	2	4	5
1	2	5	3	4
2	1	4	5	3
1	3	2	4	5
1	2	5	3	4
2	3	1	4	5
1	2	3	4	5

# processing times for each operation on a route
29	9	49	62	44
43	75	69	46	72
91	39	90	12	45
81	71	9	85	22
14	22	26	21	72
84	52	48	47	6
46	61	32	32	30
31	46	32	19	36
76	76	85	40	26
85	61	64	47	90
78	36	11	56	21
90	11	28	46	30
85	74	10	89	33
95	99	52	98	43
6	61	69	49	53
2	95	72	65	25
37	13	21	89	55
86	74	88	48	79
69	51	11	89	74
13	7	76	52	45

# job specifications, jobs have to be ordered by release date
jobs

#number of jobs
20

# <route number> <release date> <due date> <weight>
1	0	0	1
2	0	0	1
3	0	0	1
4	0	0	1
5	0	0	1
6	0	0	1
7	0	0	1
8	0	0	1
9	0	0	1
10	0	0	1
11	0	0	1
12	0	0	1
13	0	0	1
14	0	0	1
15	0	0	1
16	0	0	1
17	0	0	1
18	0	0	1
19	0	0	1
20	0	0	1
