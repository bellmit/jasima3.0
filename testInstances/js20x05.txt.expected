testInstances/js20x05.txt IGF[SPT]	simtime	1267.0
testInstances/js20x05.txt IGF[SPT]	cMax	1267.0
testInstances/js20x05.txt IGF[SPT]	flowtimes	785.25	247.00	1267.00	336.43	15705.00	20
testInstances/js20x05.txt IGF[SPT]	tardiness	785.25	247.00	1267.00	336.43	15705.00	20
testInstances/js20x05.txt IGF[SPT]	lateness	785.25	247.00	1267.00	336.43	15705.00	20
testInstances/js20x05.txt IGF[FCFS]	simtime	1645.0
testInstances/js20x05.txt IGF[FCFS]	cMax	1645.0
testInstances/js20x05.txt IGF[FCFS]	flowtimes	1277.10	715.00	1645.00	261.32	25542.00	20
testInstances/js20x05.txt IGF[FCFS]	tardiness	1277.10	715.00	1645.00	261.32	25542.00	20
testInstances/js20x05.txt IGF[FCFS]	lateness	1277.10	715.00	1645.00	261.32	25542.00	20
testInstances/js20x05.txt IGF[PTPlusWINQPlusNPT]	simtime	1335.0
testInstances/js20x05.txt IGF[PTPlusWINQPlusNPT]	cMax	1335.0
testInstances/js20x05.txt IGF[PTPlusWINQPlusNPT]	flowtimes	754.40	245.00	1335.00	326.20	15088.00	20
testInstances/js20x05.txt IGF[PTPlusWINQPlusNPT]	tardiness	754.40	245.00	1335.00	326.20	15088.00	20
testInstances/js20x05.txt IGF[PTPlusWINQPlusNPT]	lateness	754.40	245.00	1335.00	326.20	15088.00	20
testInstances/js20x05.txt IGF[WINQ]	simtime	1513.0
testInstances/js20x05.txt IGF[WINQ]	cMax	1513.0
testInstances/js20x05.txt IGF[WINQ]	flowtimes	975.50	304.00	1513.00	351.02	19510.00	20
testInstances/js20x05.txt IGF[WINQ]	tardiness	975.50	304.00	1513.00	351.02	19510.00	20
testInstances/js20x05.txt IGF[WINQ]	lateness	975.50	304.00	1513.00	351.02	19510.00	20
testInstances/js20x05.txt IGF[INV_FCFS]	simtime	1365.0
testInstances/js20x05.txt IGF[INV_FCFS]	cMax	1365.0
testInstances/js20x05.txt IGF[INV_FCFS]	flowtimes	815.70	370.00	1365.00	323.75	16314.00	20
testInstances/js20x05.txt IGF[INV_FCFS]	tardiness	815.70	370.00	1365.00	323.75	16314.00	20
testInstances/js20x05.txt IGF[INV_FCFS]	lateness	815.70	370.00	1365.00	323.75	16314.00	20
testInstances/js20x05.txt IGF[INV_SPT]	simtime	1631.0
testInstances/js20x05.txt IGF[INV_SPT]	cMax	1631.0
testInstances/js20x05.txt IGF[INV_SPT]	flowtimes	1221.50	618.00	1631.00	290.98	24430.00	20
testInstances/js20x05.txt IGF[INV_SPT]	tardiness	1221.50	618.00	1631.00	290.98	24430.00	20
testInstances/js20x05.txt IGF[INV_SPT]	lateness	1221.50	618.00	1631.00	290.98	24430.00	20
testInstances/js20x05.txt IGF[WSPT]	simtime	1267.0
testInstances/js20x05.txt IGF[WSPT]	cMax	1267.0
testInstances/js20x05.txt IGF[WSPT]	flowtimes	785.25	247.00	1267.00	336.43	15705.00	20
testInstances/js20x05.txt IGF[WSPT]	tardiness	785.25	247.00	1267.00	336.43	15705.00	20
testInstances/js20x05.txt IGF[WSPT]	lateness	785.25	247.00	1267.00	336.43	15705.00	20
testInstances/js20x05.txt IGF[INV_WSPT]	simtime	1631.0
testInstances/js20x05.txt IGF[INV_WSPT]	cMax	1631.0
testInstances/js20x05.txt IGF[INV_WSPT]	flowtimes	1221.50	618.00	1631.00	290.98	24430.00	20
testInstances/js20x05.txt IGF[INV_WSPT]	tardiness	1221.50	618.00	1631.00	290.98	24430.00	20
testInstances/js20x05.txt IGF[INV_WSPT]	lateness	1221.50	618.00	1631.00	290.98	24430.00	20
testInstances/js20x05.txt IGF[EDD]	simtime	1645.0
testInstances/js20x05.txt IGF[EDD]	cMax	1645.0
testInstances/js20x05.txt IGF[EDD]	flowtimes	1277.10	715.00	1645.00	261.32	25542.00	20
testInstances/js20x05.txt IGF[EDD]	tardiness	1277.10	715.00	1645.00	261.32	25542.00	20
testInstances/js20x05.txt IGF[EDD]	lateness	1277.10	715.00	1645.00	261.32	25542.00	20
testInstances/js20x05.txt IGF[FASFS]	simtime	1645.0
testInstances/js20x05.txt IGF[FASFS]	cMax	1645.0
testInstances/js20x05.txt IGF[FASFS]	flowtimes	1277.10	715.00	1645.00	261.32	25542.00	20
testInstances/js20x05.txt IGF[FASFS]	tardiness	1277.10	715.00	1645.00	261.32	25542.00	20
testInstances/js20x05.txt IGF[FASFS]	lateness	1277.10	715.00	1645.00	261.32	25542.00	20
testInstances/js20x05.txt IGF[SLK]	simtime	1501.0
testInstances/js20x05.txt IGF[SLK]	cMax	1501.0
testInstances/js20x05.txt IGF[SLK]	flowtimes	1212.90	697.00	1501.00	245.25	24258.00	20
testInstances/js20x05.txt IGF[SLK]	tardiness	1212.90	697.00	1501.00	245.25	24258.00	20
testInstances/js20x05.txt IGF[SLK]	lateness	1212.90	697.00	1501.00	245.25	24258.00	20
testInstances/js20x05.txt IGF[INV_CR]	simtime	1526.0
testInstances/js20x05.txt IGF[INV_CR]	cMax	1526.0
testInstances/js20x05.txt IGF[INV_CR]	flowtimes	1238.15	697.00	1526.00	232.71	24763.00	20
testInstances/js20x05.txt IGF[INV_CR]	tardiness	1238.15	697.00	1526.00	232.71	24763.00	20
testInstances/js20x05.txt IGF[INV_CR]	lateness	1238.15	697.00	1526.00	232.71	24763.00	20
testInstances/js20x05.txt IGF[INV_CR_V1]	simtime	1392.0
testInstances/js20x05.txt IGF[INV_CR_V1]	cMax	1392.0
testInstances/js20x05.txt IGF[INV_CR_V1]	flowtimes	785.15	227.00	1392.00	351.02	15703.00	20
testInstances/js20x05.txt IGF[INV_CR_V1]	tardiness	785.15	227.00	1392.00	351.02	15703.00	20
testInstances/js20x05.txt IGF[INV_CR_V1]	lateness	785.15	227.00	1392.00	351.02	15703.00	20
testInstances/js20x05.txt IGF[INV_CR_V2]	simtime	1526.0
testInstances/js20x05.txt IGF[INV_CR_V2]	cMax	1526.0
testInstances/js20x05.txt IGF[INV_CR_V2]	flowtimes	1238.15	697.00	1526.00	232.71	24763.00	20
testInstances/js20x05.txt IGF[INV_CR_V2]	tardiness	1238.15	697.00	1526.00	232.71	24763.00	20
testInstances/js20x05.txt IGF[INV_CR_V2]	lateness	1238.15	697.00	1526.00	232.71	24763.00	20
testInstances/js20x05.txt IGF[ATC(k=1.0)]	simtime	1267.0
testInstances/js20x05.txt IGF[ATC(k=1.0)]	cMax	1267.0
testInstances/js20x05.txt IGF[ATC(k=1.0)]	flowtimes	785.25	247.00	1267.00	336.43	15705.00	20
testInstances/js20x05.txt IGF[ATC(k=1.0)]	tardiness	785.25	247.00	1267.00	336.43	15705.00	20
testInstances/js20x05.txt IGF[ATC(k=1.0)]	lateness	785.25	247.00	1267.00	336.43	15705.00	20
testInstances/js20x05.txt Bremen1	simtime	1256.0
testInstances/js20x05.txt Bremen1	cMax	1256.0
testInstances/js20x05.txt Bremen1	flowtimes	733.75	245.00	1256.00	331.05	14675.00	20
testInstances/js20x05.txt Bremen1	tardiness	733.75	245.00	1256.00	331.05	14675.00	20
testInstances/js20x05.txt Bremen1	lateness	733.75	245.00	1256.00	331.05	14675.00	20
testInstances/js20x05.txt Bremen2	simtime	1317.0
testInstances/js20x05.txt Bremen2	cMax	1317.0
testInstances/js20x05.txt Bremen2	flowtimes	763.05	294.00	1317.00	317.81	15261.00	20
testInstances/js20x05.txt Bremen2	tardiness	763.05	294.00	1317.00	317.81	15261.00	20
testInstances/js20x05.txt Bremen2	lateness	763.05	294.00	1317.00	317.81	15261.00	20
testInstances/js20x05.txt IGF[IFTMinusUITPlusNPT]	simtime	1332.0
testInstances/js20x05.txt IGF[IFTMinusUITPlusNPT]	cMax	1332.0
testInstances/js20x05.txt IGF[IFTMinusUITPlusNPT]	flowtimes	776.35	302.00	1332.00	321.28	15527.00	20
testInstances/js20x05.txt IGF[IFTMinusUITPlusNPT]	tardiness	776.35	302.00	1332.00	321.28	15527.00	20
testInstances/js20x05.txt IGF[IFTMinusUITPlusNPT]	lateness	776.35	302.00	1332.00	321.28	15527.00	20
testInstances/js20x05.txt XWINQ	simtime	1662.0
testInstances/js20x05.txt XWINQ	cMax	1662.0
testInstances/js20x05.txt XWINQ	flowtimes	1031.80	304.00	1662.00	408.78	20636.00	20
testInstances/js20x05.txt XWINQ	tardiness	1031.80	304.00	1662.00	408.78	20636.00	20
testInstances/js20x05.txt XWINQ	lateness	1031.80	304.00	1662.00	408.78	20636.00	20
